ios/CodePush/CodePushUpdateUtils.m

```diff
@@ -127,6 +154,76 @@ + (NSString *)findMainBundleInFolder:(NSString *)folderPath
     return nil;
 }
 
++ (NSString *)getAssetsFolderName
+{
+    return AssetsFolderName;
+}
+
++ (NSString *)getHashForBinaryContents:(NSURL *)binaryBundleUrl
+                                 error:(NSError **)error
+{
+    if (!didLoadBinaryHash) {
+        didLoadBinaryHash = YES;
```

Comment:
Doesn't the JS side already cache the results of calling getConfiuration the first time it does? If so, we wouldn't need to cache this on the native side as well. I'd still love to think about how to cache this in user defaults as well, so we don't have to recompile this every time the app is started.